src/directives/formly-form.test.js

```diff
@@ -952,5 +952,31 @@ describe('formly-form', () => {
       expect(listener).to.have.been.called
       expect(expression).to.have.been.called
     })
+
+    it(`should setup any watchers specified on a fieldgroup`, () => {
+      scope.model = {}
+
+      const listener = sinon.spy()
+      const expression = sinon.spy()
+
+      scope.fields = [{
+        watcher: [{
+          listener: '',
+          expression: '',
+        }, {
+          listener,
+          expression,
+        }],
+        key: 'group',
```

Comment:
@kentcdodds I added a key to the fieldGroup here, otherwise I get:

```
Firefox 43.0.0 (Ubuntu 0.0.0) formly-form field watchers should setup any watchers specified on a fieldgroup FAILED
	expected [Function: compileAndDigest] to not throw an error but 'TypeError: field.key is undefined' was thrown
	AssertionError@/home/benoror/github/angular-formly/node_modules/karma-sinon-chai/node_modules/chai/chai.js:5033:18
	[3]</module.exports/Assertion.prototype.assert@/home/benoror/github/angular-formly/node_modules/karma-sinon-chai/node_modules/chai/chai.js:206:1
	assertThrows@/home/benoror/github/angular-formly/node_modules/karma-sinon-chai/node_modules/chai/chai.js:1629:1
	[10]</module.exports/ctx[name]@/home/benoror/github/angular-formly/node_modules/karma-sinon-chai/node_modules/chai/chai.js:3748:18
	@/home/benoror/github/angular-formly/src/index.test.js:16699:8
```

Not sure why since key is marked as optional